contract/src/msgs/data_requests/state/mod.rs

```diff
@@ -10,6 +11,17 @@ use timeouts::Timeouts;
 /// Governance-controlled timeout configuration parameters.
 pub const TIMEOUT_CONFIG: Item<TimeoutConfig> = Item::new("timeout_config");
 
+/// Stores the amount staked, and the staker address.
+#[cw_serde]
+pub struct StakedFunds {
```

Comment:
I think using `staked` is maybe a little confusing, given that executor nodes also stake on the contract. Maybe `escrow`?